configs/config-example.json

```diff
@@ -1,74 +1,110 @@
-{
-    "version": "1.0",
-    "zookeepers": [ "zookeeper" ],
-    "clusterId": "iudx-catalogue-cluster",
-    "modules": [
-        {
-            "id": "iudx.catalogue.server.database.DatabaseVerticle",
-            "optionalModules": [
-                 "iudx.catalogue.server.geocoding.GeocodingVerticle",
-                 "iudx.catalogue.server.nlp.NLPSearchVerticle"
-            ],
-            "verticleInstances": 2,
-            "databaseIP": "127.0.0.1",
-            "docIndex": "cattestindex",
-            "databaseUser": "user",
-            "databasePassword": "password",
-            "bypassAuth": true,
-            "databasePort": 9200
-        },
-        {
-            "id": "iudx.catalogue.server.authenticator.AuthenticationVerticle",
-            "host": "catalogue.iudx.io",
-            "verticleInstances": 2,
-            "authServerHost": "auth.iudx.org.in",
-            "jwtIgnoreExpiry": true,
-            "publicKey": "<public key of auth server>"
-        },
-        {
-            "id": "iudx.catalogue.server.validator.ValidatorVerticle",
-            "databaseIP": "127.0.0.1",
-            "databasePort": 9201,
-            "docIndex": "cattestindex",
-            "databaseUser": "user",
-            "databasePassword": "password",
-            "verticleInstances": 2
-        },
-        {
-            "id": "iudx.catalogue.server.apiserver.ApiServerVerticle",
-            "host": "catalogue.iudx.io",
-            "ssl": false,
-            "httpPort": 8080,
-	    "keystore": "configs/keystore.jks",
-	    "keystorePassword": "password",
-            "ip": "127.0.0.1",
-            "catAdmin": "datakaveri.org/h7e844e2e832398d238928abcd64f3266afa41dc",
-            "verticleInstances": 2
-
-        },
-        {
-            "id": "iudx.catalogue.server.geocoding.GeocodingVerticle",
-            "peliasUrl": "pelias_api",
-            "peliasPort": 4000,
-            "verticleInstances": 2
-        },
-        {
-            "id": "iudx.catalogue.server.nlpsearch.NLPSearchVerticle",
-            "nlpServiceUrl": "es-vectorised-search_web_1",
-            "nlpServicePort": 5000,
-            "verticleInstances":2
-        },
-	    {
-            "id": "iudx.catalogue.server.auditing.AuditingVerticle",
-            "verticleInstances": 1,
-            "auditingDatabaseIP": "",
-            "auditingDatabasePort": 5432,
-            "auditingDatabaseName": "defaultdb",
-            "auditingDatabaseUserName": "immudb",
-            "auditingDatabasePassword": "password",
-            "auditingDatabaseTableName": "tableName",
-            "auditingPoolSize": 25
-            }
-
-    ]
-}
+{

```

Comment:
Hey,  thie contains lot of test config options, which is not needed in actual deployment. You can maybe split it into another called ``config-test.json`` which as test config options and one ``config-example.json`` which does not.